Confus/Confus/Maze.h

```diff
@@ -32,6 +32,11 @@ namespace Confus
 		/// </summary>
 		int m_MazeSizeY;
 
+		/// <summary>
+		/// This Scalar increases the scale of the walls and the position that that walls are from each other thus making larger paths.
+		/// </summary>
+		float m_PathScalar;
```

Comment:
Naming sounds a bit off if it's actually changing the scale of the walls as well